plugin/src/main/java/com/playmonumenta/scriptedquests/zones/zonetree/EmptyZoneTree.java

```diff
@@ -0,0 +1,24 @@
+package com.playmonumenta.scriptedquests.zones.zonetree;
+
+import org.bukkit.util.Vector;
+
+import com.playmonumenta.scriptedquests.zones.zone.ZoneFragment;
+
+public class EmptyZoneTree extends BaseZoneTree {
+	/*
+	 * Invalidate all fragments in the tree, causing any players inside them to be
+	 * considered outside them. This updates them to the correct zone automagically.
+	 */
```

Comment:
I think it's generally bad practice to duplicate these comments a bunch - because if they get changed later, it's likely they'll be out of date in the places they were missed. I think they should only be in the interface / base abstract class.